perfkitbenchmarker/linux_benchmarks/spark_benchmark.py

```diff
@@ -31,6 +31,13 @@
 job was received by the platform and when it ran), and a runtime, which is
 the time the job took to run, as reported by the underlying cluster.
 
+This benchmark can be used to run a hadoop job by setting the spark_job_type
+flag to hadoop, eg:
```

Comment:
I'm not sure what you mean, but I edited the comments to hopefully make this clearer.